### PR TITLE
fix misuse of TLD

### PR DESCRIPTION
**Pull request by johnmuhl**

### CODE REVIEW

File: config/locales/server.en.yml [hunk 1]
```diff
@@ -980,7 +980,7 @@ en:
     fixed_category_positions: "If checked, you will be able to arrange categories into a fixed order. If unchecked, categories are listed in order of activity."
     fixed_category_positions_on_create: "If checked, category ordering will be maintained on topic creation dialog (requires fixed_category_positions)."
     add_rel_nofollow_to_user_content: "Add rel nofollow to all submitted user content, except for internal links (including parent domains). If you change this, you must rebake all posts with: \"rake posts:rebake\""
-    exclude_rel_nofollow_domains: "A list of domains where nofollow should not be added to links. tld.com will automatically allow sub.tld.com as well. As a minimum, you should add the top-level domain of this site to help web crawlers find all content. If other parts of your website are at other domains, add those too."
```
Comment (tgxworld) [inline config/locales/server.en.yml]: I don't think we need to remove `top-level` here?

Comment (johnmuhl) [inline config/locales/server.en.yml]: The `top-level domain of this site` is going to be `com`, `net`, `org` etc and I don't think the instructions mean for you to add `com` to this field; using the term `top-level` seems confusing to me. Perhaps `..add the main domain of this site...` would be more clear.

Comment (tgxworld) [inline config/locales/server.en.yml]: Ah I see what you mean now. Thanks for explaining.

**Comment by tgxworld** [comment]
Thank you for the PR @johnmuhl

**Pull request merged**